.gitignore

```diff
@@ -33,13 +25,15 @@ tests/proguard-project.txt
 
 # Android Studio and Gradle specific entries
 .gradle
-.idea
```

Comment:
Why do you remove this?

Reply:
[L46-L48](https://github.com/owncloud/android/blob/master/.gitignore#L43-L45) has more relevance, there is all excluded, except code format